composer.json

```diff
@@ -25,7 +25,7 @@
         "phpstan/phpstan-phpunit": "^0.12.18",
         "rector/rector-cakephp": "^0.10.4",
         "rector/rector-doctrine": "^0.10.6",
-        "rector/rector-installer": "^0.10.1",
+        "rector/extension-installer": "^0.10.2",
```

Comment:
@sabbelasichon Btw, I flipped the name to `extension-installer`, as `rector-<x>` name is used for framework-specific sets. Like `rector-nette`, `rector-symfony` etc.
This should make also clear it does not install rector, but extensions.